agenta-web/src/components/Playground/ViewNavigation.tsx

```diff
@@ -34,6 +35,7 @@ const ViewNavigation: React.FC<Props> = ({
         isParamSaveLoading,
         saveOptParams,
     } = useVariant(appName, variant)
+    const [isParamsCollapsed, setIsParamsCollapsed] = useState("1")
```

Comment:
why not use the boolean literal `true` here?

Reply:
isParamsCollapsed is actually used as activeKey which expects either a list or string containing the active keys.

Reply:
The right way would be to probably create another hook in the subcomponent that changes to "" or "1" when this changes I guess